### ISSUE TITLE
12 and/or 24 hours clock

### ISSUE THREAD

**Issue by marminatix** [blynkkk/beta#56]
Hi.
Europeans usually use 24 hours clock system, sometimes both 12 and 24 hours clock but Superchart X-Axis and downloaded reports seems to be in AM/PM system only and don't know how to toggle (and if it's possibile...).
Thanks.
M

**Comment by marminatix** [comment]
The charts retain the AM/PM notation but the chart reports are in 24 hours format.
For me, it's enough.
Thanks a lot.
M

**Comment by doom369** [comment]
It should depend on the user locale, this is something will be introduced soon.

**Comment by alexkipar** [comment]
@doom369 if i'm not missing anything, chart reports are not android app functionality

**Comment by doom369** [comment]
@marminatix coudl you please clarify do you mean web or Android?

**Comment by marminatix** [comment]
@doom369 
Hi.
The issue is intended ONLY for the webpage of the beta version.
The chart on the Webpage has the X-Axis with the AM/PM format only. 
Some days ago I tried to download the chart .csv report and the data format on the file was AM/PM, so I opened this issue.
Today I tried again and, without changing anything about the local settings, the format is now 24 hours.
And this is beautiful if you only think about an automated csv - to xls - to graph (or other) analisys.

The BETA android version chart doesn't have the report exporting option like the old android app but it would be a VERY appreciated feature to think about....

M
